### PR TITLE
early July - Easily fix errors on your connections

### PR DESCRIPTION
**Pull request by meganelepalud-akeneo**

### CODE REVIEW

File: content/updates/2020-07/1-easily-fix-errors-on-your-connections.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Easily fix errors on your connections 
+::: meta-data type="Improvement" features="Connections, Productivity" available="early-July" link-to-doc="../articles/connection-dashboard.html#in-the-connection-monitoring-page"
```
Comment (NolwennP) [inline content/updates/2020-07/1-easily-fix-errors-on-your-connections.md:2]:
```suggestion
::: meta-data type="Improvement" features="Connections, Productivity" available="early July" link-to-doc="../articles/connection-dashboard.html#in-the-connection-monitoring-page"
```

File: content/updates/2020-07/1-easily-fix-errors-on-your-connections.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+:::
+
+We improved the connection error messages by adding several information that should help you to fix them.
+
+In the `Monitoring` page, you will now find links to other PIM page or help center pages, and some data about the related product.  
+
+![Enriched error message](../img/fix-errors-on-your-connections.png)
```
Comment (NolwennP) [inline content/updates/2020-07/1-easily-fix-errors-on-your-connections.md:9]: The image is more than 793px wide. Also, text on the screenshot is cropped, is it on purpose?

File: content/updates/2020-07/1-easily-fix-errors-on-your-connections.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Easily fix errors on your connections
```
Comment (NolwennP) [inline content/updates/2020-07/1-easily-fix-errors-on-your-connections.md:1]: Don't forget the yaml at the beginning of the file to generate the PIM updates in the comm panel. You can find an example of the format in master (see old updates). :)

File: content/updates/2020-07/1-easily-fix-errors-on-your-connections.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Easily fix errors on your connections 
+::: meta-data type="Improvement" features="Connections, Productivity" available="early-July" link-to-doc="../articles/connection-dashboard.html#in-the-connection-monitoring-page"
+:::
+
+We improved the connection error messages by adding several information that should help you to fix them.
```
Comment (NolwennP) [inline content/updates/2020-07/1-easily-fix-errors-on-your-connections.md:5]:
```suggestion
We improved the connection error messages by adding information that will help you to fix them.
```

File: content/updates/2020-07/1-easily-fix-errors-on-your-connections.md (hunk 3 above)
Comment (meganelepalud-akeneo) [reply content/updates/2020-07/1-easily-fix-errors-on-your-connections.md:1]: Yes, I created the PR before our Tuesday meeting :) I'll add it at the beginning of the page

**Comment by meganelepalud-akeneo** [comment]
I close this PR because I was on the July-updates branch, and it will be easier to create another branch and cherry-pick my commits that merge every changes

**Pull request closed without merging**